### DISCUSSION TITLE
[Config Support]: UI timezone

### DISCUSSION THREAD

**Discussion by RN-Say** [blakeblackshear/frigate#13702] [Config Support] [answered]
### Describe the problem you are having

I'm not sure what options are acceptable for timezone In my frigate.yaml. I have tried the following (one at a time) without seeing any changes in the UI:
`timezone: America/Denver` 
`timezone: Europe/Athens`
`timezone: Asia/Nicosia`

My Home Assistant instance is set to (GMT +2:00) Nicosia

However, my laptop is set to Eastern time US and Canada and my alert review in Home Assistant shows Eastern time unless I change the time zone of my laptop. I access my home assistant through the Nabu Casa external access. FWIW, the clips under the Home Assistant media folder all show the correct local time(Nicosia). 

I _am_ able to change the format of the time display on my laptop with:
`time_format: 24hour`

Without changing frigate.yaml, I took two screen shots with the laptop set to different time zones:
![Laptop Eastern Time](https://github.com/user-attachments/assets/6575dd2e-6331-48b2-b9d3-fb0dd4acc870)
![Laptop Athens Time](https://github.com/user-attachments/assets/781524e5-e070-489a-8113-a2609163d01c)

Is this a misconfiguration or a bug?

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
ui:
  # timezone: America/Denver
  timezone: Europe/Athens
  time_format: 24hour
```

### Relevant Frigate log output

```shell
none
```

### Relevant go2rtc log output

```shell
none
```

### Frigate stats

```json
none
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

Coral

### Screenshots of the Frigate UI's System metrics pages

<img width="1703" alt="image" src="https://github.com/user-attachments/assets/e2a1048f-2d18-45c2-a0fc-748092f58e5a">

**Comment by NickM-27** [comment]
it depends where you are looking in the UI, but yeah the review item timestamp there does not seem to respect the timezone config

**Comment by RN-Say** [comment]
The above screenshots were taken from the review area under Frigate in Home Assistant. 

<img width="206" alt="image" src="https://github.com/user-attachments/assets/b0c8e4bb-056d-4d8d-851c-1e96744b3450">

**Comment by hawkeye217** [comment, marked as answer]
Fixed in https://github.com/blakeblackshear/frigate/pull/13703
